tests/python/frontend/pytorch/test_forward.py

```diff
@@ -3437,6 +3437,16 @@ def test_fn(x, weights=None):
     verify_trace_model(test_fn, [inp, weights], targets)
 
 
+def test_hard_swish():
+    hard_swish = torch.nn.Hardswish()
+    hard_swish_inplace = torch.nn.Hardswish(inplace=True)
+    examples = [torch.rand(8), torch.rand(8, 8), torch.rand(8, 16), torch.rand(1, 1, 8)]
+    targets = ["llvm", "cuda"]
+    for input in examples:
+        verify_trace_model(hard_swish, [input], targets)
```

Comment:
`verify_trace_model` is supposed to be used for dynamic models that require VM compilation. Please use `verify_model` instead.

Reply:
ok